tests/test_api.py

```diff
@@ -94,8 +94,31 @@ async def test_api_get_all_checks(setup_catalog, client, query, fake_check):
     assert second["error"] == "no-can-do"
 
 
-async def test_api_get_resource(db, client, insert_fake_resource):
-    await insert_fake_resource(db)
```

Comment:
The diff in github is a bit weird, but it makes it look like this line is gone? Is that on purpose?